ASIM/dev/ASimTester/filteringTest/filteringTest.py

```diff
@@ -0,0 +1,191 @@
+import unittest
+import os
+import sys
+import yaml
+import contextlib
+from datetime import datetime, timedelta, timezone
+from azure.monitor.query import LogsQueryClient, LogsQueryStatus
+from azure.identity import DefaultAzureCredential
+from azure.core.exceptions import HttpResponseError
+from azure.identity import DefaultAzureCredential
+from schemasParameters import all_schemas_parameters
+
+
+REPO_PATH = f"{os.getcwd()}/../../../../"
+DUMMY_VALUE = "\'!not_REAL_vAlUe\'"
+DAYS_DELTA = 729
+
+
+if len(sys.argv) == 3:
+    ws_id = sys.argv[1]
+    parser_file_relative_path = sys.argv[2]
+else:
+    print("Please provide the correct number of arguments")
+    exit(-1)
+
+end_time = datetime.now(timezone.utc)
```

Comment:
I assume this is in a next phase, but just to make sure we remember, defaulting to max would be very slow in some scenarios. We shuold start with a shorter time period and expand.